packages/components/inputs/localized-multiline-text-input/src/localized-multiline-text-input.spec.js

```diff
@@ -272,3 +283,43 @@ describe('when the error is on the selected language', () => {
     expect(getByText('Some error')).toBeInTheDocument();
   });
 });
+
+describe('when the additionalInfo exists on the selected locale should display', () => {
+  it('when given localized data', () => {
+    const { getByLabelText, getByText, queryByLabelText } =
+      renderLocalizedMultilineTextInput({
+        additionalInfo: {
+          en: 'Some description',
+        },
+      });
+    expect(getByLabelText('EN')).toBeInTheDocument();
+    expect(queryByLabelText('FR')).not.toBeInTheDocument();
+    expect(getByText('Some description')).toBeInTheDocument();
+  });
+  it('when given localized data', () => {
+    const { getByLabelText, getByText, queryByLabelText } =
+      renderLocalizedMultilineTextInput({
+        additionalInfo: {
+          en: { id: 'i18en', defaultMessage: 'english i18n message' },
+          fr: { id: 'i18fr', defaultMessage: 'french i18n message' },
+        },
+      });
+    expect(getByLabelText('EN')).toBeInTheDocument();
+    expect(queryByLabelText('FR')).not.toBeInTheDocument();
+    expect(getByText('english i18n message')).toBeInTheDocument();
+    expect(queryByLabelText('french i18n message')).not.toBeInTheDocument();
+  });
+  it('when given localized data', () => {
```

Comment:
Please, avoid repeating test names 🙏 
```suggestion
  it('when given localized data as react element', () => {
```